src/Kernel/client/kernel.ts

```diff
@@ -124,9 +124,23 @@ class Kernel {
         IPython.notebook.kernel.events.on("kernel_ready.Kernel", args => {
             this.requestEcho();
             this.requestClientInfo();
+            this.setupMyMessageListener(); //TODO: call this something else
```

Comment:
Looks like a remaining TODO here as well.

Reply:
Renamed it!